tests/e2e/solclient/env.go

```diff
@@ -0,0 +1,47 @@
+package solclient
+
+import (
+	"github.com/smartcontractkit/helmenv/environment"
+)
+
+// NewChainlinkSolOCRv2 returns a cluster config with Solana test validator
+func NewChainlinkSolOCRv2() *environment.Config {
+	return &environment.Config{
+		NamespacePrefix: "chainlink-sol",
+		Charts: environment.Charts{
+			"solana-validator": {
+				Index: 1,
+			},
+			"mockserver-config": {
+				Index: 2,
+			},
+			"mockserver": {
+				Index: 3,
+			},
+			"chainlink": {
+				Index: 4,
+				Values: map[string]interface{}{
+					"replicas": 5,
+					"chainlink": map[string]interface{}{
+						"image": map[string]interface{}{
+							"image":   "public.ecr.aws/chainlink/chainlink",
```

Comment:
This can be overridden by env var because we are using https://github.com/kelseyhightower/envconfig, it depends on our CI pipeline, if we choose network-centric CI we need to add a GHA hook to override that.